### ISSUE TITLE
Help text isn't working on subcommands

### ISSUE THREAD

**Issue by karth295** [dmtucker/yaas#26]
```
usenpalank3m1:yaas karthik$ yaas host --help
usage: yaas host [-h]

optional arguments:
  -h, --help  show this help message and exit
```

It should show the help texts of subcommands like `list` and `show`.

**Comment by karth295** [comment]
Will be fixed by https://github.com/dmtucker/yaas/pull/29

**Comment by dmtucker** [comment]
@karth295 if you mention an issue in the PR, it will close itself when the PR merges. I.e.:

```
Fixes #26. blah blah blah describe describe describe
```
